Readme.md

```diff
@@ -3,33 +3,40 @@
 
 ![alt text](./Icons/Icon_small.png)
 
-## Install
-See The following user guide for installation instructions and an introduction to Cytnx:
+[API Documentation](https://kaihsinwu.gitlab.io/cytnx_api/)
 
-[https://kaihsinwu.gitlab.io/Cytnx_doc/install.html](https://kaihsinwu.gitlab.io/Cytnx_doc/install.html)
+## What is Cytnx (pronounced as sci-tens)?
 
-## Intro slide
-[Cytnx_v0.5.pdf (dated 07/25/2020)](https://drive.google.com/file/d/1vuc_fTbwkL5t52glzvJ0nNRLPZxj5en6/view?usp=sharing)
+Cytnx is a library designed for Quantum/classical Physics simulations.
```

Comment:
Cytnx is a tensor network library designed for quantum physics simulations using tensor network algorithms.